jenkins/debian/Dockerfile_debian

```diff
@@ -0,0 +1,28 @@
+FROM debian:stretch
+
+ARG DEBIAN_ARCH=armhf
+
+RUN dpkg --add-architecture $DEBIAN_ARCH
+RUN apt-get update
+RUN if [ "$DEBIAN_ARCH" != "i386" -a "$DEBIAN_ARCH" != "amd64" ]; then  \
+        apt-get install -y --no-install-recommends crossbuild-essential-$DEBIAN_ARCH; \
+    else                                                                \
+        apt-get install -y --no-install-recommends build-essential;     \
+    fi
+RUN apt-get install --no-install-recommends -y libpciaccess-dev:$DEBIAN_ARCH \
+                       libkmod-dev:$DEBIAN_ARCH \
+                       libprocps-dev:$DEBIAN_ARCH \
+                       libcairo2-dev:$DEBIAN_ARCH \
+                       libunwind-dev:$DEBIAN_ARCH \
+                       libudev-dev:$DEBIAN_ARCH \
```

Comment:
These packages only seem to be needed to build a specific test suite. Couldn't be passed as an arg, and specified in the test-suite-specific JenkinsFile?

I'm concerned about people having to modify this file when adding new test suites, and slowing down the build process for all the others.